examples/foc/foc_fixed16_thr.c

```diff
@@ -185,9 +185,10 @@ int foc_fixed16_thr(FAR struct foc_ctrl_env_s *envp)
   struct foc_mq_s         handle;
   struct foc_motor_b16_s  motor;
   struct foc_device_s     dev;
-  int                     time      = 0;
-  int                     ret       = OK;
+  int                     time = 0;
```

Comment:
Maybe
```suggestion
#ifdef FOC_STATE_PRINT_PRE
  int                     time = 0;
#endif
```
instead?

Reply:
Can't since time is also used in other place:
https://github.com/apache/incubator-nuttx-apps/pull/1426/files#diff-df4ad1889816b80fd16d4fa0906888c2f9313974ee14e02acd88231f569c7afcR234
https://github.com/apache/incubator-nuttx-apps/pull/1426/files#diff-df4ad1889816b80fd16d4fa0906888c2f9313974ee14e02acd88231f569c7afcR372